internal/documentdb/pool.go

```diff
@@ -237,6 +238,45 @@ func (p *Pool) Collect(ch chan<- prometheus.Metric) {
 		prometheus.CounterValue,
 		float64(stats.MaxIdleDestroyCount()),
 	)
+
+	err := p.WithConn(func(conn *pgx.Conn) error {
+		// Query for all feature counter stats
+		rows, err := conn.Query(todoCtx,
+			`SELECT feature_name, usage_count FROM documentdb_api_internal.command_feature_counter_stats($1)`,
```

Comment:
We normally use generated code such as https://github.com/FerretDB/FerretDB/blob/main/internal/documentdb/documentdb_api_internal/documentdb_api_internal.go#L2456/. I assume generated code this case cannot be used because it's `QueryRow` instead of `Query`?